db/music_pg.sql

```diff
@@ -0,0 +1,373 @@
+CREATE TABLE "submission" (
+  "submission_id" uuid PRIMARY KEY DEFAULT (gen_random_uuid()),
+  "participant_id" int NOT NULL,
+  "bracket_id" int NOT NULL,
+  "song_id" int NOT NULL,
+  "order" int NOT NULL,
+  "date_submitted" timestamp NOT NULL DEFAULT (now()),
+  "active" bool_int NOT NULL,
+  "is_dupe" bool_int NOT NULL,
+  "title" text NOT NULL,
+  "version" text,
+  "artist" text,
+  "collaborators" text,
+  "composer" text,
+  "link" text NOT NULL,
+  "genre" text,
+  "emoji" text,
+  "comments" text
+);
+
+CREATE TABLE "submission_history" (
+  "submission_history_id" uuid PRIMARY KEY DEFAULT (gen_random_uuid()),
+  "date_update" timestamp NOT NULL,
```

Comment:
Very minor personal preference, for multiple related fields I like to use a prefix to indicate that they're related, e.g. here I would go with `update_date` and `update_note` so that 1) it's more obvious at first glance that they're related, 2) they end up next to each other alphabetically (if that matters somewhere) and 3) it's easier to parse when they're stacked (in general, in this case the non-prefix/suffix part is the same length), so if you had something like
```
  "comment_update"
  "note_update"
```
that's a little harder to parse and immediately see what's going on than
```
  "update_comment"
  "update_date"
```

All that being said, again this is very minor and just my own personal preference, but I figure we ought to try to align on even these sorts of conventions early on so that we aren't using different formats throughout, as then it's harder to guess what some property name might be.

Reply:
The reason I went with a suffix instead of a prefix is (again, personal preference) that I like having "date" first so that I can easily autocomplete if I'm looking for a date and don't remember what the field is called. This might just be a product of using bad autocomplete features.

Reply:
Going with update_date and update_note.